api/envoy/config/filter/network/mongo_proxy/v2/mongo_proxy.proto

```diff
@@ -25,4 +25,8 @@ message MongoProxy {
   // and KillCursors. Once an active delay is in progress, all incoming
   // data up until the timer event fires will be a part of the delay.
   envoy.config.filter.fault.v2.FaultDelay delay = 3;
+
+  // Flag to specify whether dynamic metadata should be published.
```

Comment:
Link to metadata section in filter docs?

Reply:
Addressed in 862e657.